### PR TITLE
Risk flag for custom ballot strategy and remove risk flag from funding history

### LINKED ISSUE #613: Include risk flag for Custom Reconfiguration Strategy
## Description

Using a custom reconfiguration strategy (ballot contract) may allow a project some opportunity to mislead contributors who aren't familiar with the contract. At a minimum, the project could operate with the same functionality as "No strategy", which is currently tied to a risk flag.

## Proposal

Create a new risk flag for projects using a custom reconfiguration strategy. Note: There may be very good reasons for using a custom strategy (like connecting on-chain voting to reconfigurations) that could make the project even _more_ trustless than using a standard delay strategy, so we should emphasize that any risk to contributors is uncertain.

### LINKED ISSUE #1378: `history` tab shouldn't have risk indicator
On rinkeby project `http://localhost:3000/#/v2/p/4275`, we show the (!) icon.

The icon shouldn't ever be rendered on the `history` tab.

Screen Shot 2022-07-11 at 2 02 36 PM

### PR DESCRIPTION
**Pull request by johnnyd-eth**
## What does this PR do and why?

- Risk flag for custom strategy (closes #613)
Screen Shot 2022-07-26 at 1 00 22 pm

- Remove risk flag from funding history (closes #1378)
Screen Shot 2022-07-26 at 1 09 59 pm

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by johnnyd-eth** [comment]
@peripheralist mind checking this one out?

File: src/constants/fundingWarningText.ts [hunk 1]
```diff
@@ -4,7 +4,8 @@ export const RESERVED_RATE_WARNING_THRESHOLD_PERCENT = 90
 
 export type FundingCycleRiskFlags = {
   duration: boolean
-  ballot: boolean
+  noBallot: boolean
```
Comment (peripheralist) [inline src/constants/fundingWarningText.ts:7]: nice 👌

Comment (peripheralist) [inline src/constants/fundingWarningText.ts:7]: nice 👌

File: src/components/v2/V2Project/V2FundingCycleSection/index.tsx [hunk 2]
```diff
@@ -70,9 +70,15 @@ export default function V2FundingCycleSection({
     return <NoFundingCycle />
   }
 
-  const tabText = ({ text }: { text: string }) => {
+  const tabText = ({
+    text,
+    hideRiskFlag,
```
Comment (peripheralist) [inline src/components/v2/V2Project/V2FundingCycleSection/index.tsx:75]: solid

File: src/components/v2/V2Project/V2FundingCycleSection/FundingCycleDetails.tsx [hunk 3]
```diff
@@ -329,8 +333,11 @@ export default function FundingCycleDetails({
           :
         </span>{' '}
         <FundingCycleDetailWarning
-          showWarning={unsafeFundingCycleProperties.ballot}
-          tooltipTitle={riskWarningText.ballot}
+          showWarning={
```
Comment (peripheralist) [inline src/components/v2/V2Project/V2FundingCycleSection/FundingCycleDetails.tsx:336]: maybe we could change `FundingCycleDetailWarning` to just accept a single `{warning: string}` argument where the tooltip is shown if `warning !== undefined` ? could clean up these ternary values here and in FundingAttributes.tsx,

**Review by peripheralist — APPROVED** [review]
nice @johnnyd-eth !

small suggestions but not critical so going ahead and approving. thank you ser 🙏

File: src/components/v2/V2Project/V2FundingCycleSection/FundingCycleDetails.tsx (hunk 3 above)
Comment (johnnyd-eth) [reply src/components/v2/V2Project/V2FundingCycleSection/FundingCycleDetails.tsx:336]: Very good call, will make a housekeeping issue for it

**Pull request merged**